chapters/th/chapter3/1.mdx

```diff
@@ -0,0 +1,21 @@
+<FrameworkSwitchCourse {fw} />
+
+# บทนำ
+
+ใน [Chapter 2](/course/chapter2) เราได้เรียนรู้วิธีการใช้ tokenizers และโมเดลที่ผ่านการเทรนมาแล้ว (pretrained models) ในการทำนาย แต่ถ้าเราต้องการจะใช้ dataset ของเราเองในการ fine-tune โมเดลล่ะ? นั่นคือหัวข้อของบทนี้เลย! คุณจะได้เรียนรู้:
```

Comment:
Just out of curiosity, are you keeping "pretrained models" in English because there isn't a good equivalent in Thai? One alternative I've seen in other languages is to translate the technical terms and then put the English equivalent afterwards in parentheses, e.g. 

```
TRANSLATED_TERMS (ENGLISH_TERMS เป็นภาษาอังกฤษ)
```

Reply:
Hi, lewtun

I've translated the word 'pretrained models' to 'โมเดลที่ผ่านการเทรนมาแล้ว' , so the format is just like the example you gave above. 
And to answer your question, Yes, I think 'โมเดลที่ผ่านการเทรนมาแล้ว' is not exact equivalent to 'pretrained models' as the word 'pretrained' have some underlying meanings which I can't translate all of them in one word.